src/platforms/rust/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -50,13 +50,13 @@ fn main_span2() {
 
 The Sentry SDK offers an integration for the `tower` ecosystem which can automatically continue a trace from an incoming HTTP request.
 
+When combining both layers, take care of the ordering of both. For example with tower::ServiceBuilder, always define the Hub layer before the Http one, like so:
```

Comment:
A small suggestion to simplify wording. Could we also add a sentence explaining _why_ users need to define the hub layer first? Unless it's obvious, but it's not to me :D

```suggestion
When combining both layers, order matters. For example, with tower::ServiceBuilder, you must define the Hub layer before the Http one, like so:
```

Reply:
I'm not sure about the reasoning. @shanamatthews